src/datepicker/datepicker-i18n.ts

```diff
@@ -34,9 +30,23 @@ export abstract class NgbDatepickerI18n {
 
 @Injectable()
 export class NgbDatepickerI18nDefault extends NgbDatepickerI18n {
-  getWeekdayShortName(weekday: number): string { return WEEKDAYS_SHORT[weekday - 1]; }
+  private weekdaysShort: Array<string>;
```

Comment:
If you'll be fixing `FormStyle.Format`, could you also prefix these with `_` as we usually do with private members